nix/frontend.nix

```
@@ -21,7 +21,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   pnpmDeps = pnpm.fetchDeps {
     inherit (finalAttrs) pname version src;
-    hash = "sha256-gTeGcV7zwZhIQIZ99Szet1tdKWVdEyPlXmilgZfJ/ko=";
+    hash =
+      if lib.versionAtLeast pnpm.version "9.12.1"
```

```suggestion
      if lib.versionAtLeast pnpm.version "9.12.2"
```
Sorry I've messed that up. I've only looked at the search.nixos.org, and it probably still was on the outdated commit. The hash broke with 9.12.2 and not 9.12.1.